kbs/test/data/policy_4.rego

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
maybe we should delete the empty lines

Reply:
The GO OPA engine is very picky about the format of the policy and would often fail if there were any extra lines. This test is supposed to make sure the Regorus is more resilient. I added a comment to the policy file because it does look odd.